package/gluon-core/luasrc/lib/gluon/upgrade/010-primary-mac

```diff
@@ -145,6 +151,9 @@ local primary_addrs = {
 		{'ramips', 'mt7621', {
 			'dir-860l-b1',
 		}},
+		{'ramips', 'mt76x8', {
+			'wavlink,wl-wn577a2',
+		}},
```

Comment:
It doesn't make sense to have a device appear multiple times in this file. Either the primary MAC address is set up correctly by default (phy0 address is used), in which case the device should not be added to this file at all, or the primary MAC address is on another interface (add a single override to this file in that case).

Reply:
Sorry, forgot to remove superfluous entries from testing. Have to check if primary mac is actually assigned to phy0 without an entry to this file...

Reply:
Correct MAC Adress is assigned to phy0 without tinkering on this file:
https://hannover.freifunk.net/karte/#/de/map/803f5dc523d1